internal/controllers/clusterextension_controller.go

```diff
@@ -283,21 +291,20 @@ func (r *ClusterExtensionReconciler) reconcile(ctx context.Context, ext *ocv1alp
 	// to ensure exponential backoff can occur:
 	//   - Permission errors (it is not possible to watch changes to permissions.
 	//     The only way to eventually recover from permission errors is to keep retrying).
+	installedBundle = &InstalledBundle{
+		BundleMetadata: resolvedBundleMetadata,
+		Image:          resolvedBundle.Image,
+	}
```

Comment:
This should probably be moved below the `r.Applier.Apply()` call since we haven't successfully installed until after the `Apply` step has completed. We also don't want to overwrite the information of a previously successful installation.

Reply:
@joelanford asked for this to be up here, so that we can indicate a failed install, since we are about to `Apply()`